testing-overhead/src/test/java/io/opentelemetry/util/JFRUtils.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package io.opentelemetry.util;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.util.function.BiFunction;
+import io.opentelemetry.results.AppPerfResults.MinMax;
+import jdk.jfr.consumer.RecordedEvent;
+import jdk.jfr.consumer.RecordingFile;
+
+public class JFRUtils {
+
+  public static long totalLongEvents(Path jfrFile, String eventName,
```

Comment:
```suggestion
  public static long countLongEvents(Path jfrFile, String eventName,
```

Reply:
Count is incorrect, it does a **sum** of the values, not just count them. Maybe you like `sumLongEvents` better? Very subtle distinction between summing and totaling, but I can change it if you like one better.